esp32LED.h

```diff
@@ -0,0 +1,69 @@
+#ifndef esp32LED_h
+#define esp32LED_h
+
+#include "Arduino.h"
+#include <SPI.h>
+#include "LedMatrix.h"
```

Comment:
where is this file?

Reply:
Sorry, I forget I had installed https://github.com/nhatuan84/esp32-led-matrix
I'll left a comment in esp32LED.h